daemon/daemon_unix.go

```diff
@@ -1082,20 +1055,30 @@ func getDefaultBridgeIPAMConf(
 	cfg config.BridgeConfig,
 	family int,
 ) ([]*libnetwork.IpamConf, error) {
+	// Select v4/v6 config based on family
+	fixedCIDR, fixedCIDROpt := cfg.FixedCIDR, "fixed-cidr"
+	cfgBIP, cfgBIPOpt := cfg.IP, "bip"
+	cfgDefGw, cfgDefGwOpt := cfg.DefaultGatewayIPv4, "DefaultGatewayIPv4"
+	if family == netlink.FAMILY_V6 {
+		fixedCIDR, fixedCIDROpt = cfg.FixedCIDRv6, "fixed-cidr-v6"
+		cfgBIP, cfgBIPOpt = cfg.IP6, "bip6"
+		cfgDefGw, cfgDefGwOpt = cfg.DefaultGatewayIPv6, "DefaultGatewayIPv6"
+	}
```

Comment:
Why not parse these outside of this function and pass them as arguments? I'm not a big fan of these `*Opt` vars.

Reply:
I've tarted it up a bit ... any better?

Reply:
Not sure we get it any better without touching how daemon settings are organized.